src/args.js

```diff
@@ -0,0 +1,33 @@
+const minimist = require("minimist");
+
+function getArgs(argString) {
+  return minimist(argString, {
+    string: ["default-test-path", "mc", "read"],
+    default: {
+      mc: ".",
+      "default-test-path": ""
+    }
+  });
+}
+
+function getArgString(args) {
+  args = Object.assign({}, args);
+
+  delete args.mc;
+
+  const defaultPath = args["default-test-path"].split(",");
+  delete args["default-test-path"];
+
+  const includedPaths = args._;
+  delete args._;
+
+  const paths = includedPaths.length > 0 ? includedPaths : defaultPath;
+  const params = Object.keys(args).map(key => `--${key}=${args[key]}`);
+
+  return params
+    .concat(...paths)
+    .join(" ")
+    .trim();
```

Comment:
I kinda get a weird pleasure out of writing bad throw away code...

the args are no fun because there are some that need to be used by mochii and then thrown away so the others can be passed on to mochitests.